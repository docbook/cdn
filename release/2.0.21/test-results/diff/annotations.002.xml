<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Article wrapper</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Article wrapper</h2>
         </header>
         <p deltaxml:deltaV2="A">This article tests <span id="here">annotations<a class="dialog-link" href="#annotation-xxx" title="Annotation link">
                  <span class="ui-icon ui-icon-comment inline-icon"/>
               </a>
            </span>.
Annotations can point in either direction. This paragraph contains an
annotation target.</p><div deltaxml:deltaV2="B" class="content">
            <p>This article tests <span id="here">annotations<a class="dialog-link" href="#annotation-xxx" title="Annotation link">
                     <span class="ui-icon ui-icon-comment inline-icon"/>
                  </a>
               </span>.
Annotations can point in either direction. This paragraph contains an
annotation target.</p>
            <p>This paragraph asserts that it is annotated
by the annotation “there”.<a class="dialog-link" href="#annotation-xxx" title="Annotation link">
                  <span class="ui-icon ui-icon-comment inline-icon"/>
               </a>
            </p>
            <figure id="R.1.5" class="table-wrapper">
               <div class="table">
                  <table>
                     <tbody>
                        <tr>
                           <td xml:id="r1c1">R1C1<a class="dialog-link" href="#annotation-xxx" title="Annotation link">
                                 <span class="ui-icon ui-icon-comment inline-icon"/>
                              </a>
                           </td>
                           <td xml:id="r1c2">R1C2<a class="dialog-link" href="#annotation-xxx" title="Annotation link">
                                 <span class="ui-icon ui-icon-comment inline-icon"/>
                              </a>
                           </td>
                        </tr>
                     </tbody>
                  </table>
               </div>
            </figure>
         </div>
         <p deltaxml:deltaV2="A">This paragraph asserts that it is annotated
by the annotation “there”.<a class="dialog-link" href="#annotation-xxx" title="Annotation link">
               <span class="ui-icon ui-icon-comment inline-icon"/>
            </a>
         </p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><figure deltaxml:deltaV2="A" id="R.1.5" class="table-wrapper">
            <div class="table">
               <table>
                  <tbody>
                     <tr>
                        <td xml:id="r1c1">R1C1<a class="dialog-link" href="#annotation-xxx" title="Annotation link">
                              <span class="ui-icon ui-icon-comment inline-icon"/>
                           </a>
                        </td>
                        <td xml:id="r1c2">R1C2<a class="dialog-link" href="#annotation-xxx" title="Annotation link">
                              <span class="ui-icon ui-icon-comment inline-icon"/>
                           </a>
                        </td>
                     </tr>
                  </tbody>
               </table>
            </div>
         </figure><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A=B" class="annotations-list">
            <div id="annotation-xxx" class="dialog annotation-hide">
               <p>This annotation annotates the word
“annotations” in the preceding paragraph.</p>
            </div>
            <div id="annotation-xxx" class="dialog annotation-hide">
               <p>This annotation annotates the second
paragraph of this article.</p>
            </div>
            <div id="annotation-xxx" class="dialog annotation-hide">
               <p>This annotation annotates R1C1 and R1C2.</p>
            </div>
            <div id="annotation-xxx" class="dialog annotation-hide">
               <p>This annotation annotates R1C1 and R1C2.</p>
            </div>
         </div>
      </article>
      <script deltaxml:deltaV2="A=B" type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"/>
      <script deltaxml:deltaV2="A=B" type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"/>
      <script deltaxml:deltaV2="A=B" type="text/javascript" src="../../resources/js/annotation.js"/>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>