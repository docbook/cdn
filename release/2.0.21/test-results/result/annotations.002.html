<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for annotations.002.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: annotations.002.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">5</span> differences)</th><th>Actual (<span class="adiff">1</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff del">&lt;p&gt;This article tests <span class="tag">&lt;span <span class="att">id="here"</span>&gt;annotations<span class="tag">&lt;a <span class="att">class="dialog-link"</span> <span class="att">href="#annotation-xxx"</span> <span class="att">title="Annotation link"</span>&gt;
                  <span class="tag">&lt;span <span class="att">class="ui-icon ui-icon-comment inline-icon"</span>/&gt;</span>
               &lt;/a&gt;</span>
            &lt;/span&gt;</span>.
Annotations can point in either direction. This paragraph contains an
annotation target.&lt;/p&gt;</span>
         <span class="tag diff del">&lt;p&gt;This paragraph asserts that it is annotated
by the annotation “there”.<span class="tag">&lt;a <span class="att">class="dialog-link"</span> <span class="att">href="#annotation-xxx"</span> <span class="att">title="Annotation link"</span>&gt;
               <span class="tag">&lt;span <span class="att">class="ui-icon ui-icon-comment inline-icon"</span>/&gt;</span>
            &lt;/a&gt;</span>
         &lt;/p&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;figure <span class="att">class="table-wrapper"</span> <span class="att">id="R.1.5"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="table"</span>&gt;
               <span class="tag">&lt;table&gt;
                  <span class="tag">&lt;tbody&gt;
                     <span class="tag">&lt;tr&gt;
                        <span class="tag">&lt;td <span class="att">xml:id="r1c1"</span>&gt;R1C1<span class="tag">&lt;a <span class="att">class="dialog-link"</span> <span class="att">href="#annotation-xxx"</span> <span class="att">title="Annotation link"</span>&gt;
                              <span class="tag">&lt;span <span class="att">class="ui-icon ui-icon-comment inline-icon"</span>/&gt;</span>
                           &lt;/a&gt;</span>
                        &lt;/td&gt;</span>
                        <span class="tag">&lt;td <span class="att">xml:id="r1c2"</span>&gt;R1C2<span class="tag">&lt;a <span class="att">class="dialog-link"</span> <span class="att">href="#annotation-xxx"</span> <span class="att">title="Annotation link"</span>&gt;
                              <span class="tag">&lt;span <span class="att">class="ui-icon ui-icon-comment inline-icon"</span>/&gt;</span>
                           &lt;/a&gt;</span>
                        &lt;/td&gt;</span>
                     &lt;/tr&gt;</span>
                  &lt;/tbody&gt;</span>
               &lt;/table&gt;</span>
            &lt;/div&gt;</span>
         &lt;/figure&gt;</span><span class="diff txtchg">
         </span><span class="tag nodiff">&lt;div <span class="att">class="annotations-list"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="dialog annotation-hide"</span> <span class="att">id="annotation-xxx"</span>&gt;
               <span class="tag">&lt;p&gt;This annotation annotates the word
“annotations” in the preceding paragraph.&lt;/p&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;div <span class="att">class="dialog annotation-hide"</span> <span class="att">id="annotation-xxx"</span>&gt;
               <span class="tag">&lt;p&gt;This annotation annotates the second
paragraph of this article.&lt;/p&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;div <span class="att">class="dialog annotation-hide"</span> <span class="att">id="annotation-xxx"</span>&gt;
               <span class="tag">&lt;p&gt;This annotation annotates R1C1 and R1C2.&lt;/p&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;div <span class="att">class="dialog annotation-hide"</span> <span class="att">id="annotation-xxx"</span>&gt;
               <span class="tag">&lt;p&gt;This annotation annotates R1C1 and R1C2.&lt;/p&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="http://code.jquery.com/jquery-1.6.4.min.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/annotation.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff add">&lt;div <span class="att">class="content"</span>&gt;
            <span class="tag">&lt;p&gt;This article tests <span class="tag">&lt;span <span class="att">id="here"</span>&gt;annotations<span class="tag">&lt;a <span class="att">class="dialog-link"</span> <span class="att">href="#annotation-xxx"</span> <span class="att">title="Annotation link"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="ui-icon ui-icon-comment inline-icon"</span>/&gt;</span>
                  &lt;/a&gt;</span>
               &lt;/span&gt;</span>.
Annotations can point in either direction. This paragraph contains an
annotation target.&lt;/p&gt;</span>
            <span class="tag">&lt;p&gt;This paragraph asserts that it is annotated
by the annotation “there”.<span class="tag">&lt;a <span class="att">class="dialog-link"</span> <span class="att">href="#annotation-xxx"</span> <span class="att">title="Annotation link"</span>&gt;
                  <span class="tag">&lt;span <span class="att">class="ui-icon ui-icon-comment inline-icon"</span>/&gt;</span>
               &lt;/a&gt;</span>
            &lt;/p&gt;</span>
            <span class="tag">&lt;figure <span class="att">class="table-wrapper"</span> <span class="att">id="R.1.5"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="table"</span>&gt;
                  <span class="tag">&lt;table&gt;
                     <span class="tag">&lt;tbody&gt;
                        <span class="tag">&lt;tr&gt;
                           <span class="tag">&lt;td <span class="att">xml:id="r1c1"</span>&gt;R1C1<span class="tag">&lt;a <span class="att">class="dialog-link"</span> <span class="att">href="#annotation-xxx"</span> <span class="att">title="Annotation link"</span>&gt;
                                 <span class="tag">&lt;span <span class="att">class="ui-icon ui-icon-comment inline-icon"</span>/&gt;</span>
                              &lt;/a&gt;</span>
                           &lt;/td&gt;</span>
                           <span class="tag">&lt;td <span class="att">xml:id="r1c2"</span>&gt;R1C2<span class="tag">&lt;a <span class="att">class="dialog-link"</span> <span class="att">href="#annotation-xxx"</span> <span class="att">title="Annotation link"</span>&gt;
                                 <span class="tag">&lt;span <span class="att">class="ui-icon ui-icon-comment inline-icon"</span>/&gt;</span>
                              &lt;/a&gt;</span>
                           &lt;/td&gt;</span>
                        &lt;/tr&gt;</span>
                     &lt;/tbody&gt;</span>
                  &lt;/table&gt;</span>
               &lt;/div&gt;</span>
            &lt;/figure&gt;</span>
         &lt;/div&gt;</span>
         <span class="tag nodiff">&lt;div <span class="att">class="annotations-list"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="dialog annotation-hide"</span> <span class="att">id="annotation-xxx"</span>&gt;
               <span class="tag">&lt;p&gt;This annotation annotates the word
“annotations” in the preceding paragraph.&lt;/p&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;div <span class="att">class="dialog annotation-hide"</span> <span class="att">id="annotation-xxx"</span>&gt;
               <span class="tag">&lt;p&gt;This annotation annotates the second
paragraph of this article.&lt;/p&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;div <span class="att">class="dialog annotation-hide"</span> <span class="att">id="annotation-xxx"</span>&gt;
               <span class="tag">&lt;p&gt;This annotation annotates R1C1 and R1C2.&lt;/p&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;div <span class="att">class="dialog annotation-hide"</span> <span class="att">id="annotation-xxx"</span>&gt;
               <span class="tag">&lt;p&gt;This annotation annotates R1C1 and R1C2.&lt;/p&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="http://code.jquery.com/jquery-1.6.4.min.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/annotation.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><p>This article tests <span id="here">annotations<a class="dialog-link" href="#annotation-d852e0" title="Annotation link"><span class="ui-icon ui-icon-comment inline-icon"></span></a></span>.
Annotations can point in either direction. This paragraph contains an
annotation target.</p><p>This paragraph asserts that it is annotated
by the annotation “there”.<a class="dialog-link" href="#annotation-d854e0" title="Annotation link"><span class="ui-icon ui-icon-comment inline-icon"></span></a></p><figure id="R.1.5" class="table-wrapper"><div class="table"><table><tbody><tr>
  <td xml:id="r1c1">R1C1<a class="dialog-link" href="#annotation-d855e0" title="Annotation link"><span class="ui-icon ui-icon-comment inline-icon"></span></a></td>
  <td xml:id="r1c2">R1C2<a class="dialog-link" href="#annotation-d856e0" title="Annotation link"><span class="ui-icon ui-icon-comment inline-icon"></span></a></td>
</tr></tbody></table></div></figure><div class="annotations-list"><div id="annotation-d852e0" class="dialog annotation-hide"><p>This annotation annotates the word
“annotations” in the preceding paragraph.</p>
</div><div id="annotation-d854e0" class="dialog annotation-hide"><p>This annotation annotates the second
paragraph of this article.</p>
</div><div id="annotation-d855e0" class="dialog annotation-hide"><p>This annotation annotates R1C1 and R1C2.</p>
</div><div id="annotation-d856e0" class="dialog annotation-hide"><p>This annotation annotates R1C1 and R1C2.</p>
</div></div></article><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script src="../../resources/js/prism.js"></script></td><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div class="content"><p>This article tests <span id="here">annotations<a class="dialog-link" href="#annotation-d4465e0" title="Annotation link"><span class="ui-icon ui-icon-comment inline-icon"></span></a></span>.
Annotations can point in either direction. This paragraph contains an
annotation target.</p><p>This paragraph asserts that it is annotated
by the annotation “there”.<a class="dialog-link" href="#annotation-d4467e0" title="Annotation link"><span class="ui-icon ui-icon-comment inline-icon"></span></a></p><figure id="R.1.5" class="table-wrapper"><div class="table"><table><tbody><tr>
  <td xml:id="r1c1">R1C1<a class="dialog-link" href="#annotation-d4468e0" title="Annotation link"><span class="ui-icon ui-icon-comment inline-icon"></span></a></td>
  <td xml:id="r1c2">R1C2<a class="dialog-link" href="#annotation-d4469e0" title="Annotation link"><span class="ui-icon ui-icon-comment inline-icon"></span></a></td>
</tr></tbody></table></div></figure></div><div class="annotations-list"><div id="annotation-d4465e0" class="dialog annotation-hide"><p>This annotation annotates the word
“annotations” in the preceding paragraph.</p>
</div><div id="annotation-d4467e0" class="dialog annotation-hide"><p>This annotation annotates the second
paragraph of this article.</p>
</div><div id="annotation-d4468e0" class="dialog annotation-hide"><p>This annotation annotates R1C1 and R1C2.</p>
</div><div id="annotation-d4469e0" class="dialog annotation-hide"><p>This annotation annotates R1C1 and R1C2.</p>
</div></div></article><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/annotations.002.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>