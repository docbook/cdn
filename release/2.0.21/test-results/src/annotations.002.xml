<article xmlns="http://docbook.org/ns/docbook" version="5.0">
<title>Article wrapper</title>

<para>This article tests <phrase xml:id="here">annotations</phrase>.
Annotations can point in either direction. This paragraph contains an
annotation target.</para>

<annotation annotates="here"><para>This annotation annotates the word
“annotations” in the preceding paragraph.</para>
</annotation>

<annotation xml:id="there"><para>This annotation annotates the second
paragraph of this article.</para>
</annotation>

<para annotations="there">This paragraph asserts that it is annotated
by the annotation “there”.</para>

<table>
<tbody>
<tr>
  <td xml:id="r1c1">R1C1</td>
  <td xml:id="r1c2">R1C2</td>
</tr>
</tbody>
</table>

<annotation annotates="r1c1 r1c2"><para>This annotation annotates R1C1 and R1C2.</para>
</annotation>

</article>
