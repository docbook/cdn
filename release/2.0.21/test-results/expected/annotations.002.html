<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><p>This article tests <span id="here">annotations<a class="dialog-link" href="#annotation-d852e0" title="Annotation link"><span class="ui-icon ui-icon-comment inline-icon"></span></a></span>.
Annotations can point in either direction. This paragraph contains an
annotation target.</p><p>This paragraph asserts that it is annotated
by the annotation “there”.<a class="dialog-link" href="#annotation-d854e0" title="Annotation link"><span class="ui-icon ui-icon-comment inline-icon"></span></a></p><figure id="R.1.5" class="table-wrapper"><div class="table"><table><tbody><tr>
  <td xml:id="r1c1">R1C1<a class="dialog-link" href="#annotation-d855e0" title="Annotation link"><span class="ui-icon ui-icon-comment inline-icon"></span></a></td>
  <td xml:id="r1c2">R1C2<a class="dialog-link" href="#annotation-d856e0" title="Annotation link"><span class="ui-icon ui-icon-comment inline-icon"></span></a></td>
</tr></tbody></table></div></figure><div class="annotations-list"><div id="annotation-d852e0" class="dialog annotation-hide"><p>This annotation annotates the word
“annotations” in the preceding paragraph.</p>
</div><div id="annotation-d854e0" class="dialog annotation-hide"><p>This annotation annotates the second
paragraph of this article.</p>
</div><div id="annotation-d855e0" class="dialog annotation-hide"><p>This annotation annotates R1C1 and R1C2.</p>
</div><div id="annotation-d856e0" class="dialog annotation-hide"><p>This annotation annotates R1C1 and R1C2.</p>
</div></div></article><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script src="../../resources/js/prism.js"></script></body></html>